### ISSUE TITLE
new 2.12 failures: cats-effect, blaze

### ISSUE THREAD

**Issue by SethTisue** [scala/community-builds#600]
cats-effect failure started at https://scala-ci.typesafe.com/job/scala-2.12.x-integrate-community-build/1905/

the blaze failure is at https://scala-ci.typesafe.com/job/scala-2.12.x-integrate-community-build/1906/ — waiting for 1907 to finish to find out if the problem is intermittent

**Comment by SethTisue** [comment]
```
[cats-effect] [error] /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.9/project-builds/cats-effect-f9b8c514b3322ad9547920ab23e9fb646631e3b5/core/shared/src/main/scala/cats/effect/Sync.scala:155: value catsDataMonadForStateT is not a member of object cats.data.StateT
[cats-effect] [error]       StateT.catsDataMonadForStateT[F, S].tailRecM(a)(f)
[cats-effect] [error]              ^
[cats-effect] [error] /home/jenkins/workspace/scala-2.12.x-integrate-community-build/target-0.9.9/project-builds/cats-effect-f9b8c514b3322ad9547920ab23e9fb646631e3b5/core/shared/src/main/scala/cats/effect/Sync.scala:155: value catsDataMonadForStateT is not a member of object cats.data.StateT
[cats-effect] [error]       StateT.catsDataMonadForStateT[F, S].tailRecM(a)(f)
[cats-effect] [error]              ^
[cats-effect] [error] one error found
```

**Comment by SethTisue** [comment]
```
[blaze] [error] Failed tests:
[blaze] [error] 	org.http4s.blaze.util.ActorSpec
[blaze] [error] Error during tests:
[blaze] [error] 	org.http4s.blaze.pipeline.stages.SerializingStageSpec
```

**Comment by SethTisue** [comment]
in a local test, the cats-effect failure happened, but the blaze one didn't

**Comment by SethTisue** [comment]
blaze didn't recur in 1907, so that's one intermittent. the cats-effect one is still happening

EDIT: blaze issue reported upstream at https://github.com/http4s/blaze/issues/97

**Comment by SethTisue** [comment]
possible culprit: typelevel/cats@5bb0252057156f85a8a17fb197f0f2497be3e85e

**Comment by SethTisue** [comment]
cats-effect fixed by #602

**Comment by SethTisue** [comment]
@djspiewak heads up that cats-effect will need a change to accommodate typelevel/cats@5bb0252057156f85a8a17fb197f0f2497be3e85e
